docs/user/hbase/install.rst

```diff
@@ -17,6 +17,26 @@ and untar it somewhere convenient:
     $ ls
     bin/  conf/  docs/  examples/  lib/  LICENSE.txt  logs/
 
+.. _hbase_deploy_distributed_runtime:
+
+Deploying the GeoMesa HBase distributed runtime jar
+---------------------------------------------------
+
+GeoMesa uses an HBase custom filter to improve processing of CQL queries.  In order to use the custom filter, you must
```

Comment:
(minor) I think we could stand to be more vague here.  HBase filters and co-processors are going to provide lots of functionality.